cmake/Platform/Common/Install_common.cmake

```diff
@@ -448,11 +448,17 @@ endfunction()
 function(ly_setup_runtime_dependencies)
 
     # Common functions used by the bellow code
-    install(CODE
+    if(DEFINED LY_CUSTOM_COPY_FUNCTION)
+        install(CODE
+${LY_CUSTOM_COPY_FUNCTION}
+        )
+    else()
+        install(CODE
 "function(ly_copy source_file target_directory)
     file(COPY \"\${source_file}\" DESTINATION \"\${target_directory}\" FILE_PERMISSIONS ${LY_COPY_PERMISSIONS})
 endfunction()"
-    )
+        )
+    endif()
```

Comment:
I think we should follow the same pattern that we did in `ly_install_target_overrides` where we call an override function if it exist

```
if(COMMAND ly_install_code_function_override)
    ly_install_code_function_override()
else()
    install(CODE
"function(ly_copy source_file target_directory)
    file(COPY \"\${source_file}\" DESTINATION \"\${target_directory}\" FILE_PERMISSIONS ${LY_COPY_PERMISSIONS})
endfunction()"
    )
endif()

```

Reply:
Will update.